README.md

```diff
@@ -1,7 +1,52 @@
-Schema-based data manipulation and code generation
+zcbor
+=====
+
+zcbor is a low footprint [CBOR](https://en.wikipedia.org/wiki/CBOR) library in the C language that comes with a schema-driven script tool that can validate your data, or even generate code for you.
+Aside from the script, the CBOR library is a standalone library which is tailored for use in microcontrollers.
+
+The validation part of the script can also work with YAML and JSON data.
+It can e.g. validate a YAML file against a schema and convert it into CBOR.
```

Comment:
In other nordic repos we have been told not to use these type of latin abbreviations, should we consider sticking with that for this repo as well?

```suggestion
It can for example validate a YAML file against a schema and convert it into CBOR.
```